doc/develop/optimizations/footprint.rst

```diff
@@ -49,6 +49,14 @@ the running application and to provide means for debugging and error handling:
 :kconfig:option:`CONFIG_DEBUG`
   This option can be disabled for production builds
 
+Others
+******
+
+:kconfig:option:`CONFIG_DEVICE_STORE_NAME`
+  When this option is disabled, devices will not have a unique human readable
+  name (:c:member:`device.name`). When obtaining device references at runtime
+  (i.e. :c:func:`device_get_binding`), this option can't be used as name is
+  used to find devices.
```

Comment:
I think something like this would be a bit clearer:

```
If you set this option to ``n``, you will no longer be able to use
:c:func:`device_get_binding`, because it looks up devices by name.
Use :c:macro:`DEVICE_DT_GET` or another related macro instead.
```

Reply:
done, thanks